postgresql/db-common-bgwriter-status.yml

```diff
@@ -0,0 +1,58 @@
+type: db
+data:
+  query: SELECT * FROM pg_stat_bgwriter;
+  dbUrl: jdbc:postgresql://${POSTGRESQL_DB_HOST_PORT}
+  dbDriverClass: org.postgresql.Driver
+  dbUser: ${POSTGRESQL_DB_USER}
+  dbPassword: ${POSTGRESQL_DB_PASSWORD}
+  dbAdditionalConnectionParams: ${POSTGRESQL_DB_ADDITIONAL_PARAMS}
+  #  dbVerticalModel="true" - because output of SHOW STATUS command gives each attribute in its own row
+  # (1st column being attribute name, 2nd column being attribute value)
+  dbVerticalModel: true
+
+observation:
+  - name: bgwriter
+    metricNamespace: postgresql
+
+    metric:
+      - name: bgwriter.checkpoints.scheduled
+        source: checkpoints_timed
+        type: long_counter
+        label: scheduled checkpoints
+        description: Number of scheduled checkpoints that have been performed
+
+      - name: bgwriter.checkpoints.requested
+        source: checkpoints_req
+        type: long_counter
+        label: requested checkpoints
+        description: Number of requested checkpoints that have been performed
+
+      - name: bgwriter.buffers.written
+        source: buffers_checkpoint
+        type: long_counter
+        label: Checkpoint buffers
+        description: Number of buffers written during checkpoints
+
+      - name: bgwriter.buffers.clean
+        source: buffers_clean
+        type: long_counter
+        label: Clean buffers
+        description: Number of buffers written by the background writer
```

Comment:
Sounds like possibly description of some other metric? Also, should `clean` part use past tense like in other metrics (so it is `...cleaned`?